docs/api-guide.md

```diff
@@ -697,17 +689,30 @@ Put put =
         .build();
 ```
 
-#### `Delete` operation
+##### Disable implicit pre-read for `Put` operations
 
-`Delete` is an operation to delete a record specified by a primary key.
+In Consensus Commit, write operations (`Put` and `Delete`) within a transaction require pre-reading the affected records to obtain the necessary information for the writes. If these records are not explicitly read during the transaction, ScalarDB performs an `implicit pre-read` of the records before committing the transaction.
```

Comment:
```suggestion
In Consensus Commit, an application must read a record before mutating the record with `Put` and `Delete` operations to obtain the latest states of the record in case the record exists. If an application does not read the record explicitly in a transaction, ScalarDB reads the record on behalf of the application before committing the transaction. We call the feature `implicit pre-read`.
```